### ISSUE TITLE
Username link isn’t linked on profile bio until page is refreshed.

### ISSUE THREAD

**Issue by zirta** [bluesky-social/social-app#1223]
Sometimes @handles on profile bio aren’t linked until page is refreshed.

Eg. My profiles:

https://bsky.app/profile/zirta.net

https://bsky.app/profile/zirta.art

I have only observed this on web/Safari.

**Comment by pfrazee** [comment]
Yeah we really need to be capturing links at time of publish, not at read time. That said, this issue seems to be a misfire on state updates that we can fix in the meantime.

**Comment by mozzius** [comment]
Can no longer repro
